earthquakes.py

```diff
@@ -31,31 +31,73 @@ def get_data():
 
     # We need to interpret the text to get values that we can work with.
     # What format is the text in? How can we load the values?
-    return ...
+    return json.loads(text)
 
 def count_earthquakes(data):
     """Get the total number of earthquakes in the response."""
-    return ...
+    return data.get("metadata").get("count")
 
 
 def get_magnitude(earthquake):
     """Retrive the magnitude of an earthquake item."""
-    return ...
+    return [dict.get("properties").get("mag") for dict in data.get("features")]
 
 
 def get_location(earthquake):
     """Retrieve the latitude and longitude of an earthquake item."""
     # There are three coordinates, but we don't care about the third (altitude)
-    return ...
+    list_of_coords = [dict.get("geometry").get("coordinates") for dict in data.get("features")]
+    lat_lon_only = [coords[0:2] for coords in list_of_coords]
+    return lat_lon_only
 
 
 def get_maximum(data):
     """Get the magnitude and location of the strongest earthquake in the data."""
-    ...
+    #this builds an element that has magnitude, lat, lon, altitude. I used [0:3] to discard altitude. I am also using reverse Sorted to sort by the first element only (which is mag) so that I can extract lat lat afterwards
```

Comment:
If you use `[0:3]`, this will keep the first 3 elements of a list. It works here because you are slicing it afterwards (`[1:3]`). You could also reuse the result of `get_location`, where you have already discarded the altitude, which means that you wouldn't need to slice the list again!